tests/api.test.ts

```diff
@@ -119,6 +123,9 @@ describe('End to end tests for the SDK', function () {
 
     await client.api.deleteThread(thread.id);
 
+    // We have to await 5 seconds for the thread to disappear from the cache
```

Comment:
related to the cache PR but I think we should get rid of it

Reply:
Done